logstash-core/lib/logstash/instrument/periodic_poller/jvm.rb

```diff
@@ -129,9 +163,8 @@ def default_information_accumulator
         :committed_in_bytes => 0,
         :max_in_bytes => 0,
         :peak_used_in_bytes => 0,
-        :peak_max_in_bytes  => 0
+        :peak_max_in_bytes => 0
```

Comment:
As a curiosity, what does `in_bytes` mean? With GC "used" metrics, I personally feel I understand what "max bytes" means, but I don't know what "max in bytes" means. Can you help me understand?

(Side note, I see the other metrics (:max_in_bytes, etc) are already using this `in_bytes` suffix, so I won't block this PR, but just noting my confusion about the naming)